assets/js/admin/system-status.js

```diff
@@ -112,4 +113,9 @@ jQuery( function ( $ ) {
 	};
 
 	wcSystemStatus.init();
+
+	$( '#log-viewer-select' ).on( 'click', ' h2 a.page-title-action', function( e ) {
```

Comment:
Better change `e` to `evt` to avoid any conflict, we do it in other scripts in WooCommerce.